src/react.tsx

```diff
@@ -31,7 +47,19 @@ function normalizeQuery<STORE extends typeof MSTGQLStore.Type, DATA>(
 ): Query<DATA> {
   if (typeof query === "function") return query(store)
   if (query instanceof Query) return query
-  return store.query(query, variables, { fetchPolicy, raw })
+  if (typeof window === "undefined") {
```

Comment:
You sure it's enough like that? I don't know all the possible environments that support SSR, just recall seeing some more elaborate checks in various places.

Reply:
@FredyC maybe like this `(typeof process !== 'undefined') && (process.release.name === 'node')`?

Reply:
Um, is Node only environment capable of SSR? I honestly do not know.

Reply:
I think it would be better to be explicity and pass `ssr: true` as environment config (just like the http client and such)